documentation/glossary.md

```diff
@@ -0,0 +1,56 @@
+# Glossary of aviation terms
+*Openscope uses some terms specific to aviation, this glossary attempts to define them.*
+---
+## ILS
+*abbr. of* Instrument Landing System
+
+A radio antenna at the end of the runway, known as a ''localizer'', guides pilots down onto the runway for landing, via an instrument known as a *guide scope*.
+
+## Fix
+
+A pre-defined point that describes an aircraft's current position. A fix can be an airport (`EGKK`), a waypoint (`WILLO`), or other points, such as a VOR station (`GWC`).
+
+## Waypoint
+
+A fix that is used to aid in navigation, defined in a flight plan or route. Waypoints are assigned a 5-letter code, such as `WILLO` and `ABTUM`.
+
+## Flight Level
+*abbr.* FL
+
+A plane's altitude, in 100s of feet. For example, a plane at `FL350` is flying at 35,000ft.
```

Comment:
This should include details on when this notation is used in regards to altitudes.

Reply:
And to do so, we will need to add the term `Transition Altitude` if it doesn't already exist.